reflex/utils/exec.py

```diff
@@ -36,9 +37,13 @@ def run_process_and_launch_url(run_command: list[str]):
     )
 
     for line in processes.stream_logs("Starting frontend", process):
-        if "ready started server on" in line:
-            url = line.split("url: ")[-1].strip()
+        if match := re.search("ready started server on ([0-9.:]+)", line):
```

Comment:
the walrus was introduced in python 3.8, but reflex still claims to support 3.7 although we do not test with it... this would certainly require us to drop 3.7 support in the main package.

The team has been discussing dropping 3.7 for at least a month, but we haven't made a solid decision yet. Although 3.7 went end of life 2 months ago (2023-06-27), there are still some supported linux distributions that might be shipping it.

i'm not asking you do change the code here...yet, but will discuss with the team and see if the time has come to drop 3.7 support. (Personally, I'm of the mind, that if we don't test it in CI, then we don't really support it).

Reply:
There are things you cannot do without the :=, but since this isn't one of them, happy to change it.